best-practices/rails/factories/README.md

```diff
@@ -0,0 +1,11 @@
+## Ruby on Rails Factories (FactoryGirl)
+
+###### Tips
+
+* Don't use direct association in factoryGirl.
```

Comment:
no, it's about use anyone of them in simple association, like:

```ruby
factory :product do
  user
end
```

the correct is nested association, like:

factory :product do
  factory: product_with_user do
    user
  end
end
```
and `create(:product)` doesn't create a association, only calling `create(:product_with_user)` creates.